src/components/calcite-label/calcite-label.tsx

```diff
@@ -36,116 +36,82 @@ export class CalciteLabel {
   //
   //--------------------------------------------------------------------------
-  private slottedContent;
 
-  /** the input requested with the for attribute */
-  private requestedFor: string;
-
-  /** the slotted content after it has been interpreted */
-  private displayedSlottedContent: HTMLElement[];
+  private getAttributes() {
+    // spread attributes from the component to rendered child, filtering out props
+    const props = ["layout", "theme", "scale", "status"];
+    return Array.from(this.el.attributes)
+      .filter((a) => a && !props.includes(a.name))
+      .reduce((acc, { name, value }) => ({ ...acc, [name]: value }), {});
+  }
 
   //--------------------------------------------------------------------------
   //
-  //  Private Methods
+  //  Lifecycle
   //
   //--------------------------------------------------------------------------
 
-  // emit focus event and focus the requested input if available
-  private handleClick = (event: MouseEvent) => {
-    this.emitSelectedItem(event.target);
-    if (this.requestedFor) {
-      focusElement(document.getElementById(this.requestedFor));
-    } else if (this.el.querySelector("textarea")) {
-      this.el.querySelector("textarea").focus();
-    } else if (this.el.querySelector("input")) {
-      this.el.querySelector("input").focus();
-    }
-  };
-
-  // wrap slotted text nodes in span to handle spacing of inline and inline space between layouts
-  private handleSlottedContent() {
-    const nodeList = [];
-    const requestedSlottedContent = this.el.childNodes;
-    // iterate over slotted nodes and wrap text nodes in span
-    if (requestedSlottedContent) {
-      requestedSlottedContent.forEach(function (item) {
-        if (item.nodeName === "#text" && item.textContent.trim().length > 0) {
-          const node = document.createElement("span");
-          node.classList.add("calcite-label-text");
-          node.innerHTML = item.textContent.trim();
-          nodeList.push(node as HTMLSpanElement);
-        } else if (item.nodeName !== "#text") {
-          nodeList.push(item);
-        }
-      });
-    }
-    return [...Array.from(new Set(nodeList))];
+  connectedCallback() {
+    const status = ["invalid", "valid", "idle"];
+    if (!status.includes(this.status)) this.status = "idle";
+
+    const layout = ["inline", "inline-space-between", "default"];
+    if (!layout.includes(this.layout)) this.layout = "default";
+
+    const scale = ["s", "m", "l"];
+    if (!scale.includes(this.scale)) this.scale = "m";
   }
 
-  private emitSelectedItem(target) {
-    this.calciteLabelFocus.emit({
-      labelEl: this.el,
-      interactedEl: target,
-      requestedInput: this.requestedFor
+  componentDidLoad() {
+    const labelNode = this.el.querySelector("label");
+    labelNode.childNodes.forEach((childNode) => {
+      if (childNode.nodeName === "#text" && childNode.textContent.trim().length > 0) {
+        const newChildNode = document.createElement("span");
```

Comment:
Since this is no longer using shadowRoot, can we namespace these via class like `calcite-input-text` or something so they are less likely to inherit styles on `span`?